### PR TITLE
ENG-543 publish contents widget config change on selecting contents

### PR DESCRIPTION
**Pull request by jeffgo10**
- component on selecting contents has been changed from autocomplete to regular contents table
 - took the existing component from the other widget (select a content to publish) but decoupled the content table from the modal

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **gidesan**

If they are not already assigned, you can assign the PR to them by writing `/assign @gidesan` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/widget-forms/contents-filter/ContentsFilterTable.js [hunk 1]
```diff
@@ -77,19 +78,26 @@ class ContentsFilterTable extends Component {
     onSelectAllRows(checked);
   }
 
+  get pickMode() {
```
Comment (gidesan) [inline src/ui/widget-forms/contents-filter/ContentsFilterTable.js:81]: is this getter really needed?

File: src/sass/common/form/FiltersSelectRenderer.scss [hunk 2]
```diff
@@ -60,4 +60,7 @@
     width: 10rem;
   }
 
+  &__content-table-selected {
+    margin-top: 20px;
+  }
 }
```
Comment (gidesan) [inline src/sass/common/form/FiltersSelectRenderer.scss:66]: please add trailing space

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/213#pullrequestreview-437438298" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**